### PR TITLE
Updates to MP KBs

### PR DESCRIPTION
**Pull request by brandysmith**

### CODE REVIEW

File: Marketplace/Providers/CenturyLink Marketplace Provider Program Guide.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+* The density of other Marketplace Providers already providing similar capability
+* The fee structure and pricing model for the function or feature
+* Maturity and performance of the product and company
+* Being a CenturyLink Cloud customer in good standing
+
+Applicants selected to move forward will have access to the Marketplace Integration Team to help develop an integration plan with the CenturyLink platform.
+
+The provider is responsible for completing the following steps to allow for their account to be configured as a Marketplace Provider:
+
+1) If not yet a CenturyLink Cloud customer, providers must setup a [CenturyLink Cloud](ctl.io/free-trial) account, the account is free to set up and will only accumulate charges for what you use on the account. Once becoming a certified Marketplace Provider, providers will be given access to the CenturyLink Marketplace Provider Portal, where they can accept the Marketplace Agreement, create product tiles, view their subscription reports, and more.
```
Comment (srveit) [inline Marketplace/Providers/CenturyLink Marketplace Provider Program Guide.md]: Change link to `(https://www.ctl.io/free-trial)`

File: Marketplace/Providers/CenturyLink Marketplace Provider Program Guide.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+* The user guides and applicable documentation are simple to follow, accurate and publishable to the CenturyLink Cloud knowledge base.
+
+* The solution integrates seamlessly with the CenturyLink Cloud automation framework in a manner that achieves low time-to-value in a self-service model.
+
+* The solution delivers the business value outlined in the application
+
+Candidates have up to three months to complete any integration needed, validate functionality and create the necessary documentation. During this period, CenturyLink will provide the Candidate access to a fully functioning, production cloud environment with a $2,000 USD monthly usage credit to be used for automation testing. Once the technology integration has been certified, the Partner will continue to have access to the same integration environment with the same $2,000 USD monthly integration credit in order to ensure compatibility with new features and versions. In order to continue receiving this ongoing integration benefit, Candidates will need to be a cloud customer in good standing, through standard online terms and conditions found [here](ctl.io/legal).
```
Comment (srveit) [inline Marketplace/Providers/CenturyLink Marketplace Provider Program Guide.md]: Change link to `(https://www.ctl.io/legal/)`

File: Marketplace/Providers/getting-started-guide-marketplace-ecosystem-provider-portal.md [hunk 3]
```diff
@@ -25,11 +32,21 @@ Click the **Company Information** tab. This takes you to your **Provider Details
...
 
+You can sign up to be notified upon the creation of a subscription for your software on the Company Information page by adding the appropriate email addresses to be notified in the Notification Emails field and clicking **Save**.
+
+![EPP](../../images/EPPnew3.png)
+
+The company logo, content that is provided in the **Description** field and products that are available in the CenturyLink Marketplace are what the customer will see when clicking the company name in the Marketplace.
+
+This is a sample of the customer view of your Company Information on the Marketplace:
+
+![EPP](../../images/EPPnew.png)
```
Comment (srveit) [inline Marketplace/Providers/getting-started-guide-marketplace-ecosystem-provider-portal.md]: Add the image `EPPnew.png`

File: Marketplace/Providers/marketplace-provider-guide-provider-portal-reporting.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+
+You can sign up to be notified upon the creation of a subscription for your software on the Company Information page by adding the appropriate email addresses to be notified in the Notification Emails field and clicking **Save**.
+
+![EPP](../../images/EPPnew3.png)
+
+The company logo, content that is provided in the **Description** field and products that are available in the CenturyLink Marketplace are what the customer will see when clicking the company name in the Marketplace.
+
+This is a sample of the customer view of your Company Information on the Marketplace:
+
+![EPP](../../images/EPPnew.png)
```
Comment (srveit) [inline Marketplace/Providers/marketplace-provider-guide-provider-portal-reporting.md]: Add the image file `EPPnew.png`

File: Marketplace/Providers/provider-onboarding-checklist.md [hunk 5]
```diff
@@ -10,19 +10,21 @@ The CenturyLink Marketplace allows customers to purchase software from an extens
 
 **Create CenturyLink Cloud Account (Responsible Party: Provider)**
 
-* The account can be created from [here](https://www.ctl.io/free-trial/).
+* If your company already has a CenturyLink Cloud (CLC) account please provide the Marketplace representative that you are working with your account alias. If you do not know how to find your alias, instructions are outlined [here](././determine-control-portal-alias.md).
```
Comment (srveit) [inline Marketplace/Providers/provider-onboarding-checklist.md]: Change link to `(../../Support/determine-control-portal-alias.md)`

**Comment by brandysmith** [comment]
Updated per your comments. All checks have passed. Thank you.

**Pull request merged**